ui/components/policies/policies-view.jsx

```diff
@@ -2,10 +2,16 @@ import React from 'react';
 
 import Pagers from '../pagers';
 import Policy from './policy-view';
+import ThingCounter from '../thing-counters';
 
 export default function PoliciesView({ policies, count, topicsIds }) {
+  const chunk = 25;
```

Comment:
It looks like the chunk size isn't used in `ThingCounter` -- can we remove it throughout?